camviewer_ui_impl.py

```diff
@@ -126,7 +126,7 @@ def __getattr__(self, name):
 
 
 class FilterObject(QObject):
```

Comment:
Maybe it's just me, but I found the fact that the filter installs itself on `FilterObject.__init__` really disorienting (rather than `GraphicUserInterface` instantiating then installing the filter)

I think it's less clear this way, but as they say: "when in Rome"

Reply:
Yeah, I didn't really want to refactor this more than I needed to